term-manager/impl/src/java/org/sakaiproject/acadtermmanage/dao/impl/AcademicSessionDaoImpl.java

```diff
@@ -0,0 +1,125 @@
+package org.sakaiproject.acadtermmanage.dao.impl;
+
+import java.util.List;
+
+import org.sakaiproject.acadtermmanage.dao.AcademicSessionDao;
+import org.sakaiproject.acadtermmanage.exceptions.DuplicateKeyException;
+import org.sakaiproject.acadtermmanage.logic.AcademicSessionSakaiProxy;
+import org.sakaiproject.acadtermmanage.model.Semester;
+import org.sakaiproject.coursemanagement.api.AcademicSession;
+import org.sakaiproject.coursemanagement.api.exception.IdExistsException;
+import org.sakaiproject.coursemanagement.api.exception.IdNotFoundException;
+import org.sakaiproject.exception.IdUnusedException;
+import org.sakaiproject.site.api.Site;
+
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+
+/**
+ *  A bit of an overkill: it just passes the method calls on to the Sakai proxy object.
+ *  Then again, it might be a good idea to catch the Exceptions thrown by the Coursemanagement-API 
+ *  and replace them with our own so that more general AcademicSessionDao Interface doesn't introduce an unnecessary 
+ *  dependency on the Coursemanagement-API simply because of the Exceptions this implementation uses.   
+ *  
+ *   
+ */
+@Slf4j
+public class AcademicSessionDaoImpl implements AcademicSessionDao {
+		
+	@Setter
+	private AcademicSessionSakaiProxy cm_api;
```

Comment:
remove the use of "_" and user standard camel case "cmApi".
I've seen underscores lead to errors during javadoc creation and it's not a good style.